tests/e2e/test_pgsql.py

```diff
@@ -78,19 +79,24 @@ def test_sql_workload_simple(self, ripsaw):
         log.info(f"Waiting {wait_time} seconds...")
         time.sleep(wait_time)
 
+        # Get pg_bench pod name
```

Comment:
lets use get_pod_by_pattern and remove the above sleep

Reply:
Updated. Used get_pod_by_pattern for pgbench pod name.